.vscode/extensions.json

```diff
@@ -6,6 +6,7 @@
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
     "orta.vscode-jest",
+    "streetsidesoftware.code-spell-checker"
```

Comment:
We had configuration for this extension in our workspace editor settings, but it wasn't a recommended extension for the workspace. I added this because it's a helpful and popular extension (spell check).